b2luigi/batch/processes/lsf.py

```diff
@@ -42,6 +42,14 @@ class MyLongTask(b2luigi.Task):
 
       The default is the short queue "s".
 
+    * the LSF job name can be contralled via the ``job_name`` parameter, e.g.
```

Comment:
You need an empty newline before `.. code-block`, otherwise the formatting is broken and when I build the documentation it looks like this:
![image](https://user-images.githubusercontent.com/5121824/110939401-a7674380-8335-11eb-8112-469e87a39fec.png)

```suggestion
    * the LSF job name can be contralled via the ``job_name`` parameter, e.g.
    
```

You can check by yourself by building it by running `make html` in the `docs` directory and then opening the file `_build/index.html` in your web-browser.

- [ ] fix documentation formatting